src/Normalizer/Pap/AppendCampaignStatsNormalizer.php

```diff
@@ -43,6 +47,8 @@ public function normalize($object, $format = null, array $context = [])
 
         $campaign['nb_surveys'] = $object->getCampaignHistoriesWithDataSurvey()->count();
         $campaign['nb_visited_doors'] = $this->campaignHistoryRepository->countVisitedDoors($object);
+        $campaign['nb_addresses'] = $object->getNbAddresses();
+        $campaign['nb_voters'] = $object->getNbVoters();
```

Comment:
you can add the serialization group directly in the Campaign entity and remove those two line in the normalizer